blockreassurance.php

```diff
@@ -32,6 +31,7 @@
 
 class blockreassurance extends Module implements WidgetInterface
 {
+    const POSITION_NONE = '0';
```

Comment:
Should it really need to be a string 🤔

Reply:
Yes, because Configuration::updateValue wait for in a second parameter either an array or a string

Reply:
I think it's a phpdoc problem.
If it's not a array, it convert it into array, and an array can store everything it wants.
Plus watch this line:
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Configuration.php#L453
You can store numerical value without any problem :)

Reply:
:ok: I change it :)